tools/cli/commands/connect.py

```diff
@@ -67,6 +67,15 @@
     'select *Change port > Port {}*, and start using Datalab.')
 
 
+connection_closed_message = ("""
```

Comment:
Might want to call this something like "connection_closed_format", since it's not the actual message.

Reply:
Done.